src/renderer/components/spinner/spinner.tsx

```diff
@@ -17,12 +18,9 @@ export class Spinner extends React.Component<Props, {}> {
   };
 
   render() {
-    const { center, singleColor, ...props } = this.props;
-    let { className } = this.props;
-    className = cssNames('Spinner', className, {
-      singleColor: singleColor,
-      center: center,
-    });
-    return <div {...props} className={className} ref={e => this.elem = e}/>;
+    const { center, singleColor, centerHorizontal, className, ...props } = this.props;
+    const classNames = cssNames('Spinner', className, { singleColor, center, centerHorizontal });
+
+    return <div {...props} className={classNames} ref={e => this.elem = e} />;
```

Comment:
`ref={e => this.elem = e}` can be removed.

Reply:
why?

Reply:
NVM, I see that it is not longer used. Will remove